### DISCUSSION TITLE
3 toolbar rows

### DISCUSSION THREAD

**Discussion by Iceteavanill** [xournalpp/xournalpp#5487] [Q&A]
Is it possible to have 3 toolbar rows? I like to have quick access to all the tools but when I use my tablet in portrait mode i cant have all my tools side by side. I tried to add a line to the config file but it does nothing.

![toolbars](https://github.com/xournalpp/xournalpp/assets/44265328/3ec5ebb2-363e-443b-94ce-876283b3dd54)

**Comment by rolandlo** [comment]
That's not supported.

**Comment by Ancient-Void** [comment]
I would love this. I also pretty much exclusively use my Surface in portrait mode. Having anything on the sides just eats into what little realestate there is. I Was also looking to see if I couldn't move the sidebar to the top too, but I suppose that's hard coded too.
